include/GdNeutronHPCapture.hh

```diff
@@ -20,6 +20,14 @@
 #include "G4NeutronHPChannel.hh"
 #include "G4HadronicInteraction.hh"
 
+#include "G4SystemOfUnits.hh"
```

Comment:
Why modifying neutron capture files (also below)?

Reply:
resolved in #10